Build crashes with the ngx-scrollbar dependency installed

#### Reproduction

Steps to reproduce:
1. `npm i ngx-scrollbar`
2. `npm run build:prod`

Link to the failed GitHub Actions workflow: https://github.com/livepoll/live-poll/pull/87/checks?check_run_id=1586042128
 
#### Expected Behavior

Successful build

#### Actual Behavior

Build crashes with following error message:
```
Error: node_modules/ngx-scrollbar/lib/utils/resize-sensor.directive.d.ts:1:23 - error TS2688: Cannot find type definition file for 'resize-observer-browser'.

1 /// <reference types="resize-observer-browser" />
                        ~~~~~~~~~~~~~~~~~~~~~~~
```

#### Environment

- Angular: 11.0.5
- ngx-scrollbar: 7.5.0
- Browser(s): -
- Operating System (e.g. Windows, macOS, Ubuntu): Windows

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,8 +1,8 @@
 # Changelog
 
-## 7.5.1
+## 7.5.2
 
-- fix build: Add resize-observer-browser dependency, closes [#336](https://github.com/MurhafSousli/ngx-scrollbar/issues/336) in [2d50282](https://github.com/MurhafSousli/ngx-scrollbar/commit/2d50282c4e039007ad2531ffacf7dddeeb89bda7).
+- fix build: Add resize-observer-browser dependency, closes [#336](https://github.com/MurhafSousli/ngx-scrollbar/issues/336) in [60dd480](https://github.com/MurhafSousli/ngx-scrollbar/pull/338/commits/60dd480d17949aaf656fa550589a940664cd6229).
 
 ## 7.5.0
 

--- a/projects/ngx-scrollbar/package.json
+++ b/projects/ngx-scrollbar/package.json
@@ -1,6 +1,6 @@
 {
   "name": "ngx-scrollbar",
-  "version": "7.5.1",
+  "version": "7.5.2",
   "license": "MIT",
   "homepage": "https://ngx-scrollbar.netlify.com/",
   "author": {
@@ -21,9 +21,11 @@
   ],
   "dependencies": {
     "bezier-easing": "*",
-    "resize-observer-browser": "*",
     "tslib": "^2.0.0"
-},
+  },
+  "devDependencies": {
+    "@types/resize-observer-browser": "*"
+  },
   "peerDependencies": {
     "@angular/common": ">=8.0.0",
     "@angular/core": ">=8.0.0",

